installer/modules/commons.py

```diff
@@ -78,4 +82,21 @@ def replace_string_in_file(filename,  search_string,  replace_string):
 
     with open(filename, "w") as destination:
         for line in lines:
-            destination.write(re.sub(search_string,  replace_string,  line))
\ No newline at end of file
+            destination.write(re.sub(search_string,  replace_string,  line))
+
+def log(type, message):
+    # Specify the default priority level for the logged messages. 
+    # Pass one of "emerg", "alert", "crit", "err", "warning", "notice", "info", "debug", 
+    # or a value between 0 and 7 (corresponding to the same named levels). 
+    lineprefix = '';
+    priority_levels_in_string = ["emerg", "alert", "crit", "err", "warning", "notice", "info", "debug"]
```

Comment:
It's also not needed, you do not have to lookup since you already got the value.